### PR TITLE
Closes Issue #4 Display Individual Reminder

### PR DESCRIPTION
**Pull request by madison-kerndt**
## Purpose

This pull request has the purpose of looking for feedback to route to each individual reminder. Could you please take a look at it. @martensonbj @stevekinney 

I will make more specific comments on the lines we have more pointed questions on.

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders', {  path: '/'  });
+  this.route('reminders', {  path: '/'  }, function() {
+    this.route('reminder', { path: '/:reminder_id '});
```
Comment (madison-kerndt) [inline app/router.js]: Can we talk through this path? Is the reminder the correct item to be pulling in this case? Also when is it appropriate to use a colon vs and underscore?

**Comment by madison-kerndt** [comment]
##Purpose

This pull request not only adds the information being displayed, but it also adds and active class to the chosen item, thus rendering the chosen item at the bottom of the page (highlighted in bright yellow). 

##Approach

Our approach for this feature consisted of pulling from multiple mediums including tutorials, instructors, and documentation. 

We first started by implementing the router functionality. Then we moved to the route functionality and finally the individual template. 

##Learning

I think the overall linking and structure became more clear. Specifically for me, the way in which the 'link-to' functionality finds its route became evident. 

https://guides.emberjs.com/v2.5.0/templates/links/
https://guides.emberjs.com/
linkedin.com/people/iweReconnectAction

##Test coverage

Tests were provided, they test the existence of more than 5 items rendering on the front page of the application.

##Follow-up tasks

####Before

screen shot 2016-10-12 at 4 24 29 pm

####After

http://recordit.co/ACfUmja2MH

##My feelings 
slack-imgs com

File: app/router.js (hunk 1 above)
Comment (brittanystoroz) [inline app/router.js]: Yes, the reminder is the correct path. Ember's naming conventions have a huge significance in how things are wired up in the framework. It is recognizing the plural vs. singular form of your model data, and it makes semantic/mental sense that a route for a single reminder would be called `reminder`, while the route for all reminders would map to `reminders`

Comment (brittanystoroz) [inline app/router.js]: The colon is a special operator in this case. It is needed to kick off a part of a path that is going to be dynamic/unique. The `reminder_id` is always going to be unique in this route, so the colon tells Ember that this isn't a hard-coded portion of the path, but rather we want it to change depending on what we're viewing. 

Underscores are just an unrelated, stylistic preference. You could have also written it as `/:reminderId`.

File: app/router.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders', {  path: '/'  });
+  this.route('reminders', {  path: '/'  }, function() {
+    this.route('reminder', { path: ':reminder_id'});
```
Comment (brittanystoroz) [inline app/router.js]: We still want this path to start with a `/`. Also, let's add a little extra space before the closing `});` for style and consistency.

File: app/styles/app.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.reminder-list {
+  border: 2px solid red;
+
```
Comment (brittanystoroz) [inline app/styles/app.css]: Little extra space before the closing brace on this?

**Comment by brittanystoroz** [comment]
Also, in the future, 'WIP second-issue' isn't a super helpful commit message. Even if something is a WIP, we want to be able to come up with a descriptive message about what progress the code is making, what *has* been done, even if it's not complete.

File: app/router.js (hunk 2 above)
Comment (stevekinney) [inline app/router.js]: It looks like you're missing a space at the end before the closing curly brace.

File: app/routes/reminder.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    this.get('store').find('reminder', params.reminder_id);
```
Comment (stevekinney) [inline app/routes/reminder.js]: 👍

File: app/templates/reminders.hbs [hunk 5]
```diff
@@ -1,6 +1,12 @@
-<section class="reminder">
+<section class="reminder-list">
   {{#each model as |reminder|}}
-    <article class="spec-reminder-item">{{reminder.title}}</article>
+  {{#link-to 'reminders.reminder' reminder tagName='article'}}
```
Comment (stevekinney) [inline app/templates/reminders.hbs]: It looks like the indentation is a bit off in this section.

**Comment by stevekinney** [comment]
What's the status of this?

**Comment by brittanystoroz** [comment]
Closing in favor of reviewing #22

**Pull request closed without merging**